collectors/nvd/collectors.py

```diff
@@ -210,6 +260,24 @@ def collect_updated(self) -> str:
             else ""
         )
 
+    @staticmethod
+    def create_snippet(item: dict) -> None:
+        """
+        Create a new snippet.
+        """
+        content = {
+            "cve_id": item["cve"],
```

Comment:
The feed collector uses `cve_ids`. We should maybe unify on one. It would be possible to use two but having one and create/link in `for` cycle would work for one or more items.

Reply:
I prefer to use only one too, and the `Flaw` model already uses `cve_id`.

Reply:
If a single snippet is relevant to multiple CVEs, do we create multiple identical snippets (except each having their own `cve_id`)? I don't see another way to link one snippet to multiple CVEs.

Reply:
I am for using the same naming as the flaw attributes have wherever it makes sense BUT here it seems to me as a valid exception. The thing is that a snippet can commonly result in multiple flaws and then we need to potentially store more CVE IDs. Of course the case with only a single CVE ID is very common too. The thing is that singular `cve_id` string item can only hold one CVE ID so it covers only one of the two cases but plural list item `cve_ids` can contain both one or more and therefore it covers both cases. Having two possible attributes for CVE ID, one in singular for a single one and one in plural for multiple ones seems more complicated than just having one plural holding an arbitrary number of them.

Reply:
So as we discussed, I changed `cve_id` to `cve_ids` in a snippet's content. This does not change anything for NVD because every "piece of information" will always have only one CVE id.